### PR TITLE
[Storage] Logging and Proxy Samples

### PR DESCRIPTION
**Pull request by tjprescott**
Will rebase off master once `feature/storage-preview5` branch is merged.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by rakshith91** [comment]
Thanks a lot @tjprescott 
Can you change the base branch to 
https://github.com/Azure/azure-sdk-for-python/tree/storage-samples-readme

File: sdk/storage/azure-storage-blob/samples/proxy_configuration.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+# Alternatively, proxy settings can be set using environment variables, with no
+# custom configuration necessary.
+HTTP_PROXY_ENV_VAR = 'HTTP_PROXY'
+HTTPS_PROXY_ENV_VAR = 'HTTPS_PROXY'
+os.environ[HTTPS_PROXY_ENV_VAR] = https_proxy
+
+service_client = BlobServiceClient.from_connection_string(connection_string)
+containers = list(service_client.list_containers(logging_enable=True))
+print("{} containers.".format(len(containers)))
```
Comment (rakshith91) [inline sdk/storage/azure-storage-blob/samples/proxy_configuration.py]: nitpick 101: newline

**Comment by tjprescott** [comment]
Closing in favor of PR #8186.

**Pull request closed without merging**